packages/interface/src/components/ui/Form.tsx

```diff
@@ -227,16 +230,34 @@ export const Form = <S extends z.Schema>({
     resolver: zodResolver(schema),
     mode: "onBlur",
   });
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const onSubmitForm = useCallback(
+    form.handleSubmit(
+      (data) => {
+        setErrorMessage(null);
+        onSubmit(data, form);
+      },
+      () => {
+        setErrorMessage("There are errors in the form. Please go back and check the warnings.");
+      },
+    ),
+    [setErrorMessage, onSubmit, form],
+  );
+  useEffect(() => {
+    if (errorMessage) {
+      setTimeout(() => {
+        setErrorMessage(null);
+      }, 7000);
+    }
+  }, [errorMessage]);
```

Comment:
Timeout should  be cleared and timeout value should be moved to constant.

```suggestion
  );
 
  useEffect(() => {
    if (errorMessage) {
      setTimeout(() => {
        setErrorMessage(null);
      }, 7000);
    }
  }, [errorMessage]);
```

Reply:
Done 🚀